packages/graphql-api/package.json

```diff
@@ -34,18 +34,19 @@
     "graphql-tools": "4.0.3",
     "lodash": "4.17.11",
     "nock": "10.0.1",
-    "ts-node": "7.0.1"
+    "ts-node": "7.0.1",
+    "@graphql-modules/core": "0.2.1",
+    "reflect-metadata": "0.1.12"
   },
   "peerDependencies": {
-    "graphql-tools": "^4.0.3"
+    "graphql-tools": "^4.0.3",
+    "@graphql-modules/core": "^0.2.1",
```

Comment:
is it a good idea to make it a peerDependencies if users can use the module without it? 🤔

Reply:
It's the only way to avoid multiple copies of a specific dependency, but still make sure that users know they need to install it.

Reply:
I am not a big fan of forcing users to install it if they don't need it ... Separate minor versions are not compatibles between them?

Also other question, what is `reflect-metadata`?

Reply:
`reflect-metadata` is required for DI, but now it is optional. Then I removed it from the code and introduction.